circulating_supply/src/lockup.rs

```diff
@@ -57,14 +49,18 @@ pub(super) async fn get_lockup_contract_state(
             )
         }
     };
+
     let view_state = view_state_result.values.get(0).with_context(|| {
         format!(
             "Failed to find encoded lockup contract for {}, block_height {}",
             account_id, block_height
         )
     })?;
 
-    let mut state = LockupContract::try_from_slice(&view_state.value)
+    let decoded_view_state = base64::decode(&view_state.value)
```

Comment:
`JsonRpcClient` doesn't return the state in decoded Base64 like `ViewClient` does, so we have to do it manually

Reply:
Heads up, 0.15 versions of nearcore crates will return decoded values, and it is important to drop these `decode` calls as Rust won't throw compilation errors here for you, and you will end up trying decoding raw (decoded) bytes instead of base64-encoded bytes.

Reply:
Thanks 👍🏽 I'll add a note in `Cargo.toml` so we know to remove this when we upgrade